### PR TITLE
Add security policy level work mode support

### PR DESCRIPTION
**Pull request by echkenluo**

### CODE REVIEW

File: pkg/agent/datapath/policyBridge.go [hunk 1]
```diff
@@ -436,60 +480,93 @@ func (p *PolicyBridge) RemoveLocalEndpoint(endpoint *Endpoint) error {
...
-func (p *PolicyBridge) AddMicroSegmentRule(rule *EveroutePolicyRule, direction uint8, tier uint8) (*FlowEntry, error) {
+func (p *PolicyBridge) AddMicroSegmentRule(rule *EveroutePolicyRule, direction uint8, tier uint8, mode string) (*FlowEntry, error) {
 	var ipDa *net.IP = nil
 	var ipDaMask *net.IP = nil
 	var ipSa *net.IP = nil
 	var ipSaMask *net.IP = nil
 	var err error
+	if mode == "" {
+		mode = DEFAULT_POLICY_ENFORCEMENT_MODE
+	}
```
Comment (wu-cl) [inline pkg/agent/datapath/policyBridge.go:517]: We could set default value in CRD, not compare it here.

Comment (wu-cl) [inline pkg/agent/datapath/policyBridge.go:517]: We could set default value in CRD, not compare it here.

File: pkg/agent/datapath/multiBridgeDatapath.go [hunk 2]
```diff
@@ -310,12 +320,12 @@ func (datapathManager *DpManager) InitializeDatapath(stopChan <-chan struct{}) {
 	// add rules for internalIP
 	for _, internalIP := range datapathManager.datapathConfig.InternalIPs {
 		// internal ingress rule
-		err := datapathManager.AddEveroutePolicyRule(newInternalIngressRule(internalIP), InternalIngressRulePrefix, POLICY_DIRECTION_IN, POLICY_TIER2)
+		err := datapathManager.AddEveroutePolicyRule(newInternalIngressRule(internalIP), InternalIngressRulePrefix, POLICY_DIRECTION_IN, POLICY_TIER3, "work")
```
Comment (KeyongSun) [inline pkg/agent/datapath/multiBridgeDatapath.go:323]: Use the "DEFAULT_POLICY_ENFORCEMENT_MODE" defined before.

File: pkg/agent/datapath/multiBridgeDatapath.go [hunk 3]
```diff
@@ -310,12 +320,12 @@ func (datapathManager *DpManager) InitializeDatapath(stopChan <-chan struct{}) {
...
 		if err != nil {
 			log.Fatalf("Failed to add internal whitelist: %v", err)
 		}
 		// internal egress rule
-		err = datapathManager.AddEveroutePolicyRule(newInternalEgressRule(internalIP), InternalEgressRulePrefix, POLICY_DIRECTION_OUT, POLICY_TIER2)
+		err = datapathManager.AddEveroutePolicyRule(newInternalEgressRule(internalIP), InternalEgressRulePrefix, POLICY_DIRECTION_OUT, POLICY_TIER3, "work")
```
Comment (KeyongSun) [inline pkg/agent/datapath/multiBridgeDatapath.go:328]: Ditto

Comment (KeyongSun) [inline pkg/agent/datapath/multiBridgeDatapath.go:328]: Ditto

File: pkg/agent/datapath/multiBridgeDatapath_test.go [hunk 4]
```diff
@@ -142,7 +142,7 @@ func testLocalEndpoint(t *testing.T) {
 
 func testERPolicyRule(t *testing.T) {
 	t.Run("test ER policy rule", func(t *testing.T) {
-		if err := datapathManager.AddEveroutePolicyRule(rule1, "rule1", POLICY_DIRECTION_IN, POLICY_TIER1); err != nil {
+		if err := datapathManager.AddEveroutePolicyRule(rule1, "rule1", POLICY_DIRECTION_IN, POLICY_TIER2, "work"); err != nil {
```
Comment (KeyongSun) [inline pkg/agent/datapath/multiBridgeDatapath_test.go:145]: DEFAULT_POLICY_ENFORCEMENT_MODE

Comment (KeyongSun) [inline pkg/agent/datapath/multiBridgeDatapath_test.go:145]: DEFAULT_POLICY_ENFORCEMENT_MODE

File: pkg/agent/datapath/multiBridgeDatapath_test.go [hunk 5]
```diff
@@ -156,13 +156,13 @@ func testERPolicyRule(t *testing.T) {
 			t.Errorf("Failed to remove ER policy rule, rule %v in cache", rule1)
 		}
 
-		if err := datapathManager.AddEveroutePolicyRule(rule2, "rule2", POLICY_DIRECTION_OUT, POLICY_TIER0); err != nil {
+		if err := datapathManager.AddEveroutePolicyRule(rule2, "rule2", POLICY_DIRECTION_OUT, POLICY_TIER1, "work"); err != nil {
```
Comment (KeyongSun) [inline pkg/agent/datapath/multiBridgeDatapath_test.go:159]: Ditto

Comment (KeyongSun) [inline pkg/agent/datapath/multiBridgeDatapath_test.go:159]: Ditto

File: pkg/agent/datapath/multiBridgeDatapath_test.go [hunk 6]
```diff
@@ -156,13 +156,13 @@ func testERPolicyRule(t *testing.T) {
...
 			t.Errorf("Failed to add ER policy rule: %v, error: %v", rule2, err)
 		}
 		if _, ok := datapathManager.Rules[rule2.RuleID]; !ok {
 			t.Errorf("Failed to add ER policy rule, not found %v in cache", rule2)
 		}
-		if err := datapathManager.AddEveroutePolicyRule(rule2, "rule2", POLICY_DIRECTION_OUT, POLICY_TIER0); err != nil {
+		if err := datapathManager.AddEveroutePolicyRule(rule2, "rule2", POLICY_DIRECTION_OUT, POLICY_TIER1, "work"); err != nil {
```
Comment (KeyongSun) [inline pkg/agent/datapath/multiBridgeDatapath_test.go:165]: Ditto

Comment (KeyongSun) [inline pkg/agent/datapath/multiBridgeDatapath_test.go:165]: Ditto

File: pkg/agent/datapath/multiBridgeDatapath_test.go [hunk 7]
```diff
@@ -176,8 +176,8 @@ func testFlowReplay(t *testing.T) {
 	}
 	t.Run("add ER policy rule", func(t *testing.T) {
 		Eventually(func() error {
-			log.Infof("add policy rule to datapath, tier: %d", POLICY_TIER2)
-			return datapathManager.AddEveroutePolicyRule(rule1, "rule1", POLICY_DIRECTION_IN, POLICY_TIER2)
+			log.Infof("add policy rule to datapath, tier: %d", POLICY_TIER3)
+			return datapathManager.AddEveroutePolicyRule(rule1, "rule1", POLICY_DIRECTION_IN, POLICY_TIER3, "work")
```
Comment (KeyongSun) [inline pkg/agent/datapath/multiBridgeDatapath_test.go:180]: Ditto

Comment (KeyongSun) [inline pkg/agent/datapath/multiBridgeDatapath_test.go:180]: Ditto

File: pkg/agent/datapath/policyBridge.go (hunk 1 above)
Comment (echkenluo) [reply pkg/agent/datapath/policyBridge.go:517]: ok, I get it

**Pull request merged**